experiments/AtmosGCM/heldsuarez-tracers.jl

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env julia --project
+using ClimateMachine
+using ArgParse
+
+s = ArgParseSettings()
+@add_arg_table! s begin
+    "--number-of-tracers"
+    help = "Number of dummy tracers"
+    metavar = "<number>"
+    arg_type = Int
+    default = 0
```

Comment:
@bischtob Why do we have two versions of held suarez? by default, no tracers are set in this script. If the only functional difference between this held suarez and the other one is the tracer argument, why not just add this tracer stuff to `heldsuarez.jl`?

Reply:
Works for me!